source/ui/Image.js

```diff
@@ -173,14 +180,15 @@
 		*/
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
+			var multiple= "url('"+ enyo.path.rewrite(src) +"'), url('"+ this.placeholder + "');";
 			if (this.sizing) {
-				this.applyStyle('background-image', src ? 'url(' + enyo.path.rewrite(src) + ')' : 'none');
+				this.applyStyle('background-image', src ? multiple : 'none');
 			} else {
 				if (!src) {
 					// allow us to clear the src property
 					this.setAttribute('src', '');
 				} else {
-					this.setAttribute('src', enyo.path.rewrite(src));
+					this.applyStyle('background-image', multiple );
```

Comment:
I think we have to carefully consider how we're approaching this. With the change on this line, we would always be setting the background-image on this element, and the `src` attribute would never be set. Previously, we were only setting the background-image when `sizing` had been specified, so we have to determine how the value of `placeholder` fits into this. For example, if we specify `placeholder` but not `sizing`, do we still set background-image? If so, what kind of sizing would we default to? We could use a sensible default in this scenario, i.e. `cover`, so that the entire background is covered. We just want to make sure we don't alter the existing behavior and allow the `src` property to still be set. We should also make it clear that specifying a value for `placeholder` relies on using background-image.